.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*    @circleci/mktg-web-optimizations
```

Comment:
This team name may be changing in the near future, but we can update that when/if that happens.

Reply:
Yes, I'm constantly trying to keep this up to date so let's cross the bridge when we get to it 🙂